execution_engine_testing/test_support/src/lib.rs

```diff
@@ -41,12 +41,17 @@ pub use step_request_builder::StepRequestBuilder;
 pub use upgrade_request_builder::UpgradeRequestBuilder;
 pub use wasm_test_builder::{InMemoryWasmTestBuilder, LmdbWasmTestBuilder, WasmTestBuilder};
 
+const DAY_MILLIS: u64 = 24 * 60 * 60 * 1000;
+
 /// Default number of validator slots.
 pub const DEFAULT_VALIDATOR_SLOTS: u32 = 5;
 /// Default auction delay.
 pub const DEFAULT_AUCTION_DELAY: u64 = 3;
 /// Default lock-in period of 90 days
-pub const DEFAULT_LOCKED_FUNDS_PERIOD_MILLIS: u64 = 90 * 24 * 60 * 60 * 1000;
+pub const DEFAULT_LOCKED_FUNDS_PERIOD_MILLIS: u64 = 90 * DAY_MILLIS;
+/// Default length of total vesting schedule of 91 days.
+pub const DEFAULT_VESTING_SCHEDULE_PERIOD_MILLIS: u64 = 91 * DAY_MILLIS;
+
```

Comment:
maybe I'm overthinking this, but for purposes of the test builder which is a general purpose testing tool for everyone to use, I think the DEFAULT for lock-in should be 0 days. 

our very specialized subset of tests proving that a post-genesis lock-in delay and a vesting schedule (which was a peculiarity of our mainnet launch which may or may not be relevant to other chains using the codebase) work correctly would then impute 90/91 days in situ instead of using the defaults.

Reply:
Ticketed as [#3231](https://github.com/casper-network/casper-node/issues/3231).